src/scribe_data/cli/get.py

```diff
@@ -79,44 +79,67 @@ def get_data(
     -------
         The requested data saved locally given file type and location arguments.
     """
-    # MARK: Defaults
+    # Mark: Defaults
```

Comment:
Quick note on marks, @KesharwaniArpita and @axif0: They only work when they're all caps :)